eng/vcpkg-commit.txt

```diff
@@ -1 +1 @@
-tags/2020.11
\ No newline at end of file
+fcd3a3bb53356cc954f7fa07cee2c99b253d1a49
```

Comment:
Do we end to use a sha here? Is there no tag to use instead?

Reply:
Yep, talked to vcpkg folks, and the recommendation for getting package dependencies from the registry is to use recent commit shas (that are updated at some feasible cadence we decide) and have two separate test pipelines (one for making sure the SDK builds on newer versions, and also one pinned to the oldest set of dependencies we want to support). They don't publish the tags, given the vcpkg tool and the registry are decoupled.

> I think the "maximally mature" approach would be the following:
a. You should have a build using the "oldest" set of dependencies you ostensibly support
b. You should have a build using a semi-current version that you update reasonably often (1/mo, for example)
c. You should have a smoke-test CI or automated PR that tries to grab latest. When that passes, it should merge into (b)

Reply:
OK. I just wanted to make sure we had a plan. 

One of the issues I see with using raw sha's is you don't easily have information about how old it is without going and digging into the repo itself. Perhaps we should add a comment in the file to help better document which how old the version it, or at least have that metadata in the commit message for this repo.

Reply:
https://github.com/Azure/azure-sdk-for-cpp/issues/3165